lib/zoho_auth_client.rb

```diff
@@ -0,0 +1,40 @@
+class ZohoAuthClient
+  ACCESS_TOKEN_REQUEST_ENDPOINT = 'https://accounts.zoho.com/oauth/v2/token'.freeze
+
+  CLIENT_ID = ArchiveConfig.ZOHO_CLIENT_ID
+  CLIENT_SECRET = ArchiveConfig.ZOHO_CLIENT_SECRET
+  ZOHO_REFRESH_TOKEN = ArchiveConfig.ZOHO_REFRESH_TOKEN
```

Comment:
Can you make sure `config.yml` has placeholder values for all of these settings? They should go around here: https://github.com/otwcode/otwarchive/blob/1f182644505830de9eb9548c97846456be59e66f/config/config.yml#L347-L353

I think this is all of them:
ArchiveConfig.ABUSE_ZOHO_DEPARTMENT_ID
ArchiveConfig.SUPPORT_ZOHO_DEPARTMENT_ID
ArchiveConfig.ZOHO_CLIENT_ID
ArchiveConfig.ZOHO_CLIENT_SECRET
ArchiveConfig.ZOHO_REFRESH_TOKEN
ArchiveConfig.ZOHO_ORG_ID

Reply:
The `ArchiveConfig` values that are no longer used (e.g. `ABUSE_AUTH`) should be removed from this file.